.github/workflows/reusable-pr-check-boxes-complete.yaml

```diff
@@ -0,0 +1,40 @@
+name: Check Boxes Checked in PR Description
+
+on:
+  workflow_call:
+  pull_request:
```

Comment:
Does it still work with these here too?

Reply:
Yes I think so!

Reply:
@biosafetylvl5 Actually - do we want the "pull_request" ones here?  Will this make it run twice on the geoips_ci repo (since it lives in the .github/workflows folder)?  Or maybe even 3 times... (org-wide ruleset, ruleset- workflow, and reusable- workflow)

Reply:
We're going to do an experiment and see if it does indeed run 3 times.